tools/eth-migration/metamask.html

```diff
@@ -426,7 +426,7 @@
 								"type": "uint64"
 							},
 							{
-								"name": "schemaIds",
+								"name": "intentIds",
```

Comment:
I think we need to add the new payloads in here such as `PaginatedDeleteSignaturePayloadV3` and etc and need to verify existing one for field name changes. These should also match with whatever we have onchain for eip712 compatbility.

Reply:
Done